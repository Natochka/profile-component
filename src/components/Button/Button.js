import styled from 'styled-components'

const Button = styled.button`
  border: none;
  border-radius: 25px;
  color: #fff;
  text-transform: uppercase;
  background-color: #ffa640;
  font-size: 14px;
  line-height: 18px;
  padding: 14px 20px;
  letter-spacing: 4px;
  outline: none;
  cursor: pointer;
`
export default Button
