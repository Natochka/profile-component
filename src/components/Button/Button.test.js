import React from 'react'
import { render } from 'react-testing-library'
import 'jest-dom/extend-expect'
import 'react-testing-library/cleanup-after-each'
import Button from './Button'

describe('Button component test', () => {
  test('button should be rendered correctly', () => {
    const { container } = render(<Button />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
