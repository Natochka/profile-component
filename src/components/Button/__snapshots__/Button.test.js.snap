// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component test button should be rendered correctly 1`] = `
<button
  class="sc-bdVaJa hKLKUL"
/>
`;

exports[`Button component test modal should be rendered correctly 1`] = `
<button
  class="sc-bdVaJa hKLKUL"
/>
`;
